tests/test_config.py

```diff
@@ -0,0 +1,23 @@
+import importlib
+import os
+
+import pytest
+
+from app import config
+
+
+@pytest.fixture
+def reload_config(os_environ):
```

Comment:
hmmm i'd be tempted to just bundle the `os_environ` fixture in to this. i'm not convinced we'll need to use os_environ much outside of testing the config loading itself, given we already have patterns for setting config values themselves temporarily for tests and that seems like it'll have much less potential side-effects and interesting quirks with needing to reload imports etc.

Reply:
Fair enough, I remember thinking the same. 

I think it's helpful to keep it separate to tease out the two things it does.

Changed in https://github.com/alphagov/notifications-template-preview/pull/651/commits/4fa5c3d0b0b3a7e36b52a137129144fc8ce03757.

Reply:
Nice compromise 👍